CppCoreGuidelines.md

```diff
@@ -21018,7 +21018,7 @@ However, we have had many requests for a set of naming and layout conventions to
 
 More specific and detailed rules are easier to enforce.
 
-These rules bear a strong resemblance to the recommendations in the [PPP Style Guide](http://www.stroustrup.com/Programming/PPP-style.pdf)
+These rules bear a big resemblance to the recommendations in the [PPP Style Guide](http://www.stroustrup.com/Programming/PPP-style.pdf)
```

Comment:
Worse grammar

Reply:
Any suggestion, or should I just revert?

Reply:
I chose to change structure a bit to avoid an adjective, plz look at changes

Reply:
"are very similar" works for me.